### ISSUE TITLE
Video effect causes crashing 

### ISSUE THREAD

**Issue by uzelessknowledge** [smeighan/xLights#773]
On Mac version 54 the video effects crashes if you select it to render the effect of try to add the path to the video (which I'm assuming is also trying to render on the model). Attempted the same in version 50 and the crash is not experienced.
I was using the Pentatonix Hallelujah video downloaded from youtube. Tried with existing sequence and also with a new sequence in a fresh layout with only 1 model. Crashing still present.

**Comment by keithsw1111** [comment]
Do you have a crash report or can you package logs?

**Comment by dkulp** [comment]
Looks like bad data is being sent into log4j.   Stack trace:

`
4   libsystem_c.dylib                   0x00007fffd59de9fc __vfprintf + 5701
5   libsystem_c.dylib                   0x00007fffd5a07423 __v2printf + 699
6   libsystem_c.dylib                   0x00007fffd59eb92a _vsnprintf + 586
7   libsystem_c.dylib                   0x00007fffd59eb9d3 vsnprintf + 80
8   liblog4cpp.5.dylib                  0x0000000107128d3f _ZN7log4cpp10StringUtil5vformEPKcP13__va_list_tag + 127
9   liblog4cpp.5.dylib                  0x00000001070fb269 _ZN7log4cpp8Category19_logUnconditionallyEiPKcP13__va_list_tag + 73
10  liblog4cpp.5.dylib                  0x00000001070fb84e _ZN7log4cpp8Category5debugEPKcz + 446
11  xLights                             0x00000001061ec75b _ZN11VideoEffect6RenderEP6EffectRK11SettingsMapR12RenderBuffer + 1995
12  xLights                             0x00000001063a6a96 _ZN12xLightsFrame19RenderEffectFromMapEP6EffectiiRK11SettingsMapR16PixelBufferClassRbbP11RenderEvent + 326
`

**Comment by keithsw1111** [comment]
Do you have the log so i can see the last message

**Comment by dkulp** [comment]
```
2016-12-08 22:20:29,022 123145379680256 log_base [INFO] Video loaded: /Users/coachryan/Desktop/Christmas2016/Trolls/video-cant.mp4
2016-12-08 22:20:29,022 123145379680256 log_base [INFO]       Length MS: 229771
2016-12-08 22:20:29,022 123145379680256 log_base [INFO]       _videoStream->avg_frame_rate.num: 24000
2016-12-08 22:20:29,022 123145379680256 log_base [INFO]       _videoStream->avg_frame_rate.den: 1001
2016-12-08 22:20:29,022 123145379680256 log_base [INFO]       DTS per sec: 24000
2016-12-08 22:20:29,022 123145379680256 log_base [INFO]       _videoStream->nb_frames: 5509
2016-12-08 22:20:29,032 123145379680256 log_base [INFO]       Source size: 640x360
2016-12-08 22:20:29,032 123145379680256 log_base [INFO]       Source coded size: 640x368
2016-12-08 22:20:29,033 123145379680256 log_base [INFO]       Output size: 128x96
2016-12-08 22:20:29,033 123145379680256 log_base [CRIT] Crash handler called.
```

**Comment by keithsw1111** [comment]
I reckon it is the logging of duration treatment

**Comment by keithsw1111** [comment]
I have this funny feeling the ' cause the %s to behave differently

**Comment by keithsw1111** [comment]
Thats the only thing I can think of as otherwise it looks ok. Although this is not the only place we use the ' so maybe not.

**Comment by uzelessknowledge** [comment]
[xLights_dbgrpt-12850-20161209T083526.zip](https://github.com/smeighan/xLights/files/642399/xLights_dbgrpt-12850-20161209T083526.zip)

**Comment by keithsw1111** [comment]
This was fixed in 2016.55
